app/services/callback_time_saver.rb

```diff
@@ -0,0 +1,23 @@
+class CallbackTimeSaver
+  def initialize(selected_answer_store:)
```

Comment:
As with the change to `address_saver.rb` later on, is it worth defaulting `selected_answer_store` to `SelectedAnswerStore` so we don't have to keep new-ing this up everywhere?

Reply:
Yeah, that would be preferable - the problem is that `SelectedAnswerStore` needs the `session` and I don't think that's accessible outside the Controller.

Reply:
:+1: